CHANGELOG

```diff
@@ -1,4 +1,32 @@
 # Changelog
+## [0.2.5] - 2020-02-19
+### Breaking changes
+ - Deprecate specifying keyword arguments as positional arguments in the public methods:
+   This will become a hard failure in Nmstate-0.3.0 and later.
+   - "libnmstate.apply()"
+   - "libnmstate.commit()"
+   - "libnmstate.rollback()"
+   - "libmstate.show()"
+
+ - NetworkManager 1.22.8 or later is now required.
+
+### New Features
+ - Report missing NM plugin during validation.
+ - Support OVS Link Aggregation (Bonding).
+
+### Bug Fixes
+ - Fix "AtributeError" when editing an existing OVS interface.
+ - Do not remove slaves implicitly when removing the master interface.
+ - Fix team interface port support.
+ - Prevent virtual interface removal after DHCP or IPV6-RA timeout.
+
+### Known Issues
+
+In nmstate 0.2.5 there are some known issues:
+ - All changes to bridges might disrupt their connectivity.
+ - Cannot set Linux Bond option to "arp_validate=active".
```

Comment:
This is partially correct, the issue is not really agreed upon.
Nm team argued that this value is irrelevant for some bond modes and that is why it is failing.
I would drop it.

Not sure about the next one with the fail-over.